ros/src/twist_controller/CMakeLists.txt

```diff
@@ -129,7 +129,9 @@ include_directories(
 ## Declare a C++ library
 add_library(dbw_node_core
   include/twist_controller/dbw_node_core.h
+  include/twist_controller/predicted_control_values.h
   src/dbw_node_core.cpp
+  src/predicted_control_values.cpp
```

Comment:
write a single library here, not integrated into other libs. or integrate into other small module.